mpact/models.py

```diff
@@ -30,21 +30,33 @@ def __str__(self):
 
 class Bot(models.Model):
     chat = models.OneToOneField(Chat, on_delete=models.CASCADE)
-    bot_id = models.IntegerField(unique=True)
+    bot_id = models.IntegerField()
     username = models.CharField(max_length=350)
     first_name = models.CharField(max_length=350)
     last_name = models.CharField(max_length=350, null=True)
 
+    class Meta:
+        constraints = [
+            models.UniqueConstraint(fields=["chat", "bot_id"], name="unique_chat_bot")
```

Comment:
I don't understand this change. If you can have the same bot in different chats, why not change `chat` from `OneToOne` to `ManyToMany`, and leave `bot_id` as unique?

Reply:
The first point is good, I will change it to One to many but I think we can't keep bot_id as unique because we have one to many relationships i.e., there can be more than one entry for the same bot with a different group. That's why I made two columns unique.